doc/conf.py

```diff
@@ -31,6 +31,7 @@
 # ones.
 extensions = [
     'myst_parser',
+    'sphinx.ext.linkcode',
```

Comment:
Surprised you put this above `sphinx.ext.autodoc` :stuck_out_tongue_winking_eye: